cypress/utils/index.ts

```diff
@@ -0,0 +1,66 @@
+import { fillAndCheckShippingSimulator } from './fill'
+import { openCart, closeCart, clearCart } from './cart'
+import { checkText } from './assertion'
+import {
+  scrollToId,
+  goToSearchPage,
+  goToProductPageByShelf,
+} from './navigation'
+
+export function resolveIdentifier(dataTestId: string[] | string) {
+  let identifier = ''
+
+  if (Array.isArray(dataTestId)) {
+    dataTestId = dataTestId.map((id) => {
+      return `[data-testid="${id}"]`
+    })
+
+    identifier = dataTestId.join(' ')
+  } else {
+    identifier = `[data-testid="${dataTestId}"]`
+  }
+
+  return identifier
+}
+
+/*
+  Pages: *
+  Selectors: {{ param }}
+*/
+interface IdentifierOptions {
+  index: number
+}
+
+export function identifierExists(
+  dataTestId: string[] | string,
+  options?: IdentifierOptions
+) {
+  let exists = false
+
+  if (options?.index) {
+    if (Cypress.$(resolveIdentifier(dataTestId)).eq(options?.index).length) {
+      exists = true
+    }
+
+    return exists
+  }
+
+  if (Cypress.$(resolveIdentifier(dataTestId)).length) {
+    exists = true
+  }
+
+  return exists
+}
+
+module.exports = {
```

Comment:
did you mean to use `export default` here?
```suggestion
export default {
```

Reply:
Same for [cypress.js](https://github.com/vtex/test-tools/pull/45/files#diff-b40833fa80d94de0505cd6870718ff858903a5f3b5bbe6a08dd79456de79bdf1R4)?

Reply:
`cypress.js` is in commonjs, probably to be able to import it directly from node